Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -224,7 +224,7 @@ TextureCacheBase::TCacheEntryBase* TextureCacheBase::DoPartialTextureUpdates(Tex
 
 	TexCache::iterator iter = textures_by_address.lower_bound(entry_to_update->addr);
 	TexCache::iterator iterend = textures_by_address.upper_bound(entry_to_update->addr + entry_to_update->size_in_bytes);
-	bool entry_need_scaling = true;
+	bool entry_need_scaling = g_ActiveConfig.bCopyEFBScaled;
```

Comment:
Please keep in mind that some games just copy their texture once and continue to use them for a very long time. eg fzero shadows, mkwii minimap, mii face